guides/Rakefile

```diff
@@ -7,7 +7,7 @@ namespace :guides do
   namespace :generate do
     desc "Generate HTML guides"
     task :html do
-      ruby "-Eutf-8:utf-8", "rails_guides.rb"
```

Comment:
I'm not sure if removing the encoding breaks things now.

Reply:
This is probably from before Ruby defaulted to utf8